addons/event_sale/models/sale_order.py

```diff
@@ -68,27 +66,4 @@ def _get_product_catalog_domain(self):
         :rtype: list
         """
         domain = super()._get_product_catalog_domain()
-        return expression.AND([domain, [('service_tracking', '!=', 'event')]])
-
-    def _notify_get_recipients_groups(self, message, model_description, msg_vals=None):
```

Comment:
This removal seems unrelated, I think this came from a recent PR maybe there was an issue when rebasing?

Reply:
@reth-odoo this whole file seems odd